runtime.md

```diff
@@ -22,6 +22,8 @@ This allows the hooks to perform cleanup and teardown logic after the runtime de
 * **`pid`**: (int) is the ID of the main process within the container, as seen by the host.
 * **`bundlePath`**: (string) is the absolute path to the container's bundle directory.
 This is provided so that consumers can find the container's configuration and root filesystem on the host.
+* **`Labels`**: (map) Labels holds runtime-specified information that is not structured by this specification
+The runtime could store experimental information here before it's standardized in this specification.
```

Comment:
Is there a recommended format for the values? Is it byte encoded or string or something else?